case-server/src/main/java/com/powsybl/caseserver/CaseApplication.java

```diff
@@ -16,6 +16,6 @@
 public class CaseApplication {
 
     public static void main(String[] args) {
-        SpringApplication.run(CaseApplication.class, args);
```

Comment:
Is this intended ? Now we can't pass additional args on the command line. At minimum we should concatenate with the CLI args, but isn't there a better wya to do this ?

Reply:
sorry it's an omission, transferred to application.yaml